### PR TITLE
Add mm-common and libxml++ packages

### PR DESCRIPTION
**Pull request by thcrain-msft**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Add mm-common and libxml++ packages as they are required by some of our partners.

###### Change Log  

- Add mm-common and libxml++ packages

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- Local build of affected packages

### CODE REVIEW

File: SPECS/libxml++/libxml++.spec [hunk 1]
```diff
@@ -0,0 +1,70 @@
+Summary:        libxml++
+Name:           libxml++
+Version:        3.2.0
+Release:        2%{?dist}
```
Comment (PawelWMS) [inline SPECS/libxml++/libxml++.spec:4]: Please add `Vendor` and `Distribution` tags.

File: SPECS/mm-common/mm-common.signatures.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+ "Signatures": {
+  "mm-common-1.0.0.tar.xz": "b97d9b041e5952486cab620b44ab09f6013a478f43b6699ae899b8a4da189cd4"
+ }
+}
+
+
```
Comment (PawelWMS) [inline SPECS/mm-common/mm-common.signatures.json:7]: Nit: one line can be removed.

File: SPECS/libxml++/libxml++.spec [hunk 3]
```diff
@@ -0,0 +1,70 @@
+Summary:        libxml++
```
Comment (PawelWMS) [inline SPECS/libxml++/libxml++.spec:1]: Please also update `SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md ` for both packages.

File: SPECS/libxml++/libxml++.spec [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+Name:           libxml++
+Version:        3.2.0
+Release:        2%{?dist}
+License:        LGPLv2+
+URL:            http://libxmlplusplus.sourceforge.net/
+Source0:        http://ftp.gnome.org/pub/GNOME/sources/libxml++/3.2/%{name}-%{version}.tar.xz
```
Comment (PawelWMS) [inline SPECS/libxml++/libxml++.spec:7]: Nit: let's use HTTPS and we can change one `libxml++` into `%{name}`.

File: SPECS/libxml++/libxml++.spec [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+%changelog
+* Mon Oct 12 2020 Thomas Crain <thcrain@microsoft.com> - 3.2.0-3
+- Remove .la files
+
+* Fri Jun 05 2020 Jonathan Chiu <jochi@microsoft.com> - 3.2.0-2
+- Update dependency names
+
+* Mon Mar 30 2020 Jonathan Chiu <jochi@microsoft.com> - 3.2.0-1
+- Original version for Mariner
```
Comment (PawelWMS) [inline SPECS/libxml++/libxml++.spec:70]: Please update this to `Original version for CBL-Mariner.`.

File: SPECS/libxml++/libxml++.spec [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+%{_libdir}/*.so
+%{_libdir}/pkgconfig/*
+
+%files doc
+%{_docdir}/*
+%{_datadir}/devhelp/*
+
+%changelog
+* Mon Oct 12 2020 Thomas Crain <thcrain@microsoft.com> - 3.2.0-3
+- Remove .la files
```
Comment (PawelWMS) [inline SPECS/libxml++/libxml++.spec:64]: Please also verify the license.

File: SPECS/libxml++/libxml++.spec (hunk 1 above)
Comment (PawelWMS) [reply SPECS/libxml++/libxml++.spec:4]: Same for the other spec.

File: SPECS/libxml++/libxml++.spec (hunk 4 above)
Comment (PawelWMS) [reply SPECS/libxml++/libxml++.spec:7]: Similar comment for the other spec.

File: SPECS/mm-common/mm-common.spec [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+%changelog
+* Mon Oct 12 2020 Thomas Crain <thcrain@microsoft.com> - 1.0.0-3
+- Update Source0 (removes need for libstdc++.tag file)
+- Lint for Mariner style
+
+* Tue Jun 09 2020 Jonathan Chiu <jochi@microsoft.com> - 1.0.0-2
+- Include libstdc++.tag in source files so package can be built offline
+
+* Mon Mar 30 2020 Jonathan Chiu <jochi@microsoft.com> - 1.0.0-1
+-   Original version for CBL-Mariner
```
Comment (PawelWMS) [inline SPECS/mm-common/mm-common.spec:49]: Nit: could you fix the double space?

File: SPECS/libxml++/libxml++.spec (hunk 6 above)
Comment (PawelWMS) [reply SPECS/libxml++/libxml++.spec:64]: Same for the other spec.

**Review by PawelWMS — CHANGES_REQUESTED** [review]

File: SPECS/mm-common/mm-common.spec (hunk 7 above)
Comment (thcrain-msft) [reply SPECS/mm-common/mm-common.spec]: Thanks- fixed this and all other comments on this pass.

**Review by PawelWMS — APPROVED** [review]
:shipit:

**Review by PawelWMS — CHANGES_REQUESTED** [review]
:clock1:

**Review by PawelWMS — APPROVED** [review]
:shipit:

**Pull request merged**